protocol/communities/community.go

```diff
@@ -1290,6 +1290,14 @@ func (o *Community) RequestsToJoin() []*RequestToJoin {
 	return o.config.RequestsToJoin
 }
 
+func (o *Community) ChatIDs() []string {
+	var chatIDs []string
+	for id := range o.config.CommunityDescription.Chats {
+		chatIDs = append(chatIDs, o.IDString()+id)
+	}
+	return chatIDs
+}
+
```

Comment:
This is entirely optional, but a cool thing about go is that you can declare vars in your return parameters also.

```suggestion
func (o *Community) ChatIDs() (chatIDs []string) {
	for id := range o.config.CommunityDescription.Chats {
		chatIDs = append(chatIDs, o.IDString()+id)
	}
	return chatIDs
}

```

And if you do declare your return params vars you can also lazy return:

```suggestion
func (o *Community) ChatIDs() (chatIDs []string) {
	for id := range o.config.CommunityDescription.Chats {
		chatIDs = append(chatIDs, o.IDString()+id)
	}
	return
}

```

But like I said this is completely a personal preference, I was just sharing some go coolness.